tests/e2e/specs/activate-and-setup/setup-wizard.test.js

```diff
@@ -1,35 +1,240 @@
 		await AdminFlow.goToPlugins();
 
-		deactivateLink = await page.$(
-			`tr[data-slug="${ slug }"] .deactivate a`
-		);
-		if ( deactivateLink ) {
-			return;
-		}
+		await expect( page ).toClick( '.sensei-message a', {
+			text: 'Setup Wizard',
+		} );
+		await page.waitForNavigation();
+		await expect( page ).toMatch( 'Welcome to Sensei LMS!' );
+	} );
 
-		await page.click( `tr[data-slug="${ slug }"] .activate a` );
+	describe( 'Welcome step', () => {
+		beforeAll( async () => await resetSetupWizard() );
 
-		// Go to the plugins page again to make sure it's in the plugin page.
-		// It can be the setup wizard.
-		await AdminFlow.goToPlugins();
-		deactivateLink = await page.waitForSelector(
-			`tr[data-slug="${ slug }"] .deactivate a`
-		);
+		it( 'opens on first launch', async () => {
+			await openSetupWizard();
+			await expect( page ).toMatch( 'Welcome to Sensei LMS!' );
+		} );
+
+		it( 'displays usage tracking modal when clicking continue', async () => {
+			await expect( page ).toClick( 'button', { text: 'Continue' } );
+			await expect( page ).toMatch( 'Build a Better Sensei LMS' );
+		} );
+
+		it( 'allows opting in', async () => {
+			await expect( page ).toClick( 'label', {
+				text: 'Yes, count me in!',
+			} );
+
+			await expect( page ).toClick(
+				'.sensei-setup-wizard__usage-modal button',
+				{
+					text: 'Continue',
+				}
+			);
+		} );
+		it( 'marks welcome step done and goes to purpose step', async () => {
+			await stepIsComplete( 'Welcome' );
+			await stepIsActive( 'Purpose' );
+			await expect( page ).toMatch(
+				'What is your primary purpose for offering online courses?'
+			);
+		} );
+	} );
+
+	describe( 'Purpose step', () => {
+		it( 'opens when it is the active step', async () => {
+			await openSetupWizard();
+			await stepIsComplete( 'Welcome' );
+			await stepIsActive( 'Purpose' );
+			await expect( page ).toMatch(
+				'What is your primary purpose for offering online courses?'
+			);
+		} );
+
+		it( 'disables Continue until something is selected', async () => {
+			await expect( page ).toMatchElement( 'button[disabled]', {
+				text: 'Continue',
+			} );
+		} );
+
+		it( 'allows selecting purposes', async () => {
+			await expect( page ).toClick( 'label', {
+				text: 'Promote your business',
+			} );
+			await expect( page ).toClick( 'label', { text: 'Other' } );
+			await expect( page ).toFill(
+				'.sensei-setup-wizard__textcontrol-other input',
+				'Other'
+			);
+
+			await expect( page ).toClick( 'button', { text: 'Continue' } );
+		} );
+
+		it( 'marks purpose step done and goes to features step', async () => {
+			await stepIsComplete( 'Purpose' );
+			await stepIsActive( 'Features' );
+			await expect( page ).toMatch(
+				'Enhance your online courses with these optional features.'
+			);
+		} );
+	} );
+
+	describe( 'Features step', () => {
+		beforeAll( async () => {
+			await AdminFlow.deactivatePlugin( 'sensei-media-attachments' );
```

Comment:
I think we're not doing anything with the `sensei-media-attachments`, right?

Reply:
Yup good catch, removed in 3341370